examples/time_series_forecasting.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<ul><li>typo: "corresponds to the ..."</li><li>suggestion: "time stamp" -&gt; "timestamp"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Thanks! Fixed.